modules/ec2/scripts/setup_scripts.sh

```diff
@@ -8,7 +8,7 @@ sudo usermod -aG docker ec2-user
 newgrp docker
 
 cd /home/ec2-user
-git clone -b prod https://github.com/2408-Team-3/flytrap_ui.git ui
+git clone https://github.com/2408-Team-3/flytrap_ui.git ui
```

Comment:
Should we change this to point to the new url with our new organisation name just to be sure?

Reply:
Yes, I'm updating that and outputs in a different branch